### PR TITLE
rust fmt file with the reformatting

### PR DESCRIPTION
**Pull request by StanChe**
A simple rust fmt used with +nightly

### CODE REVIEW

File: rustfmt.toml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+comment_width = 100
+edition = "2021"
+group_imports = "One"
+imports_granularity = "One"
```
Comment (kstepanovdev) [inline rustfmt.toml:4]: I'd rather go with `Module`

File: rustfmt.toml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+comment_width = 100
+edition = "2021"
+group_imports = "One"
```
Comment (kstepanovdev) [inline rustfmt.toml:3]: Is `One` even an [option](https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=group#group_imports)?

Comment (StanChe) [reply rustfmt.toml:3]: [It is](https://rust-lang.github.io/rustfmt/?version=master&search=group#group_imports) at least with nightly. Took this example from the spl library

Comment (kstepanovdev) [reply rustfmt.toml:3]: I see. Let it be.

**Pull request merged**